### PR TITLE
Change in 'plotter.py' and addition of tests in .xml format

### PR DESCRIPTION
**Pull request by robfairh**
This PR addresses issue #175 by changing the scale in the plots made by 'plotter.py' from log to linear. 

The PR addresses issue #190 by changing the labels and titles in the plots made by 'plotter.py'.

The PR also addresses issue #193 by splitting the file 'algorithm_performance_tests.py' into 4 files:
- performance_test1.ipynb
- performance_test2.ipynb
- performance_test3.ipynb
- performance_test4.ipynb

It also splits 'algorithm_performance_tests_backdeployment.py' into two files:
- performance_test5.ipynb
- performance_test6.ipynb

While 'algorithm_performance_tests.py' and 'algorithm_performance_tests_backdeployment.py' produce the cyclus input files in '.json' format, the '.ipynb' files produce them in '.xml' format. They also add the plots from the results to the files for easier visualization.

### CODE REVIEW

**Comment by gwenchee** [comment]
Looks good! i'll check out the ipynb's later

File: d3ploy/plotter.py [hunk 1]
```diff
@@ -94,26 +100,29 @@ def plot_demand_supply_agent(all_dict, agent_dict, commod, test, demand_driven):
...
+    if commod=='power':
+        ax.set_ylabel('Power (MW)')
+    else:
+        ax.set_ylabel('Mass (Kg)')
     handles, labels = ax2.get_legend_handles_labels()
     ax2.legend(handles, labels, fontsize=11, loc='upper left',
                fancybox=True)
     
-    ax1.set_title('Supply, Demand and prototypes for %s' %commod)
+    ax1.set_title('Supply, Demand and prototypes for %s' %test)
```
Comment (gwenchee) [inline d3ploy/plotter.py]: why did u change this from commod to test?

**Comment by robfairh** [comment]
I am working on the PEP 8 issues, then we can merge it.

**Comment by robfairh** [comment]
This PR also addresses issue #195 by making a few changes in 'tester.py', see https://github.com/arfc/d3ploy/pull/194/commits/050a23131ab9fcd2abd88fb531cb8d58abb56226

Now, the plot shows 'calc_demand' and 'calc_supply' completely and under_supply reflects the plot:
case6_ma_reactorout

borrar3

**Comment by robfairh** [comment]
This PR is too large. I am splitting it into several ones.
#201 is one. There are more to come.

**Comment by robfairh** [comment]
#214 makes the changes in tester.py.

The performance tests have to be rewritten after merging #213.

**Pull request closed without merging**